src/components/App/App.tsx

```diff
@@ -7,8 +8,28 @@ import MainPage from '../MainPage/MainPage';
 import GamePage from '../GamePage/GamePage';
 import EndPage from '../EndPage/EndPage';
 import ErrorPage from '../ErrorPage/ErrorPage';
+import { Question } from '../../types';
+import { getQuestions } from '../../apiCall';
 
 function App() {
+
+  const [questions, setQuestions] = useState<Question | null>(null);
+
+  const fetchData = async () => {
+    try {
+      const data = await getQuestions()
+      if (data !== "DUPLICATE DEV FETCH") {
+        setQuestions(data)
+      }
+    } catch (error) {
+      console.log("ERROR:", error)
+    }
+}
+
+  useEffect(() => {
+    fetchData()
+  }, [fetchData])
```

Comment:
I wonder if removing the fetchData from the dependency array will help with the error?